src/node/internal/streams_duplex.js

```diff
@@ -70,9 +70,22 @@ Object.setPrototypeOf(Duplex, Readable);
 }
 
 // Use the `destroy` method of `Writable`.
-Duplex.prototype.destroy = Writable.prototype.destroy;
+// Guard against module initialization race condition where Writable.prototype.destroy
```

Comment:
Can we fix the race condition itself eventually?

Reply:
Are we even actually sure there is a race condition at all?
AFAIU it reproduced deterministiclly

Reply:
Found a better fix.